### PR TITLE
Fix error navigating between profile and learner search pages

### LINKED ISSUE #3608: Error clicking on 'My Profile' link from learner search page
To reproduce:
 - be a staff user
 - go to `/learners`
 - click on 'My Profile'. You should be at your profile page.
 - click on the Micromasters link. You should be directed to the learner search page
 - open the drawer and click on 'My Profile'. You should see a stack trace and the drawer should fail to open.

### PR DESCRIPTION
**Pull request by noisecapella**
#### What are the relevant tickets?
Fixes #3608 

#### What's this PR do?
Clears the dashboard data when a user leaves learner page. This fixes the error caused by the consistency check for price calculation.

#### How should this be manually tested?
See repro steps in #3608

### CODE REVIEW

**Comment by annagav** [comment]
screen shot 2017-10-06 at 11 49 28 am

**Comment by noisecapella** [comment]
Should be fixed now. The price display on the learner page is removed

**Comment by aliceriot** [comment]
fixes the error! reading code now.

File: .flowconfig [hunk 1]
```diff
@@ -4,6 +4,7 @@
 .*/node_modules/draftjs-utils/.*
 .*/node_modules/react-event-listener/src/index.js
 .*.git/.*
+.*/staticfiles/.*
```
Comment (aliceriot) [inline .flowconfig]: :+1:

**Pull request merged**